__tests__/components/Header.test.jsx

```diff
@@ -0,0 +1,75 @@
+import '@testing-library/jest-dom'
+import { render, screen } from '@testing-library/react'
+import Page from '../../components/Header'
+import { useSession } from 'next-auth/react';
+
+jest.mock('next/dist/client/router', () => jest.requireActual('next-router-mock'))
+
+describe('Testing login authentication', () => {
+    it('Test if Login is Authenticated', () => {
+        const oneDay = 86400
+        const twoDays = 2 * oneDay
```

Comment:
What made you want to use twoDays instead of just oneDay?

Reply:
Frankly, This is a random. should I be using oneDay instead?

Reply:
I think it is better to use something as simple as possible that satisfies the testing goals. There are two constants used here when just 1 is enough.